### PR TITLE
Friendlier KAFKA_TOPIC

### PR DESCRIPTION
**Pull request by thody**
- add instructions to create config var for topic
- blow up on boot if topic env var isn't set (rather than waiting for a POST to the resource)

### CODE REVIEW

File: config.yml [hunk 1]
```diff
@@ -3,3 +3,6 @@ logging:
 
   loggers:
     com.heroku.kafka.demo: DEBUG
+
+kafka:
```
Comment (kvlou) [inline config.yml]: do you need that? i assume the configuration will just grab the env variable?

Comment (thody) [inline config.yml]: This is how Dropwizard does the mapping of the env var to `KafkaConfig.java`. Advantage of doing it this way is we can validate the var at boot time. Previously it was reading the env var at request time.

Comment (kvlou) [inline config.yml]: Cool thanks for explaining.

**Comment by halorgium** [comment]
lgtm

**Pull request merged**